### DISCUSSION TITLE
[dp compress] command giving error

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2221] [Q&A] [answered]
Hi,

Command --
```bash
dp compress -i graph.pb -o graph-compress.pb
```
System -- DGX V100
Dataset -- [H2O, PBE0TS](https://dplibrary.deepmd.net/#/project_details?project_id=202109.002)

Error --
![Screenshot 2023-01-04 at 7 05 01 PM](https://user-images.githubusercontent.com/10074710/210567625-a6d2d3b6-6033-4581-866b-013551c9ae2e.png)

**Comment by njzjz** [comment, marked as answer]
#2104
